Issue fixed: calendar positioning

Decrepidos opened the pull request:
Files changed:
CalendarMain.js
Change:
Replaced < p > wrapper with < div class="insertCalendar" >

File changed:
src/css/calendar.css
Change
Added:
div.insertCalendar {
    position: relative;
}
Updated
div.calendar {
  position: fixed; to
top:25%; to

div.calendar {
  position: absolute;
  top:30% which top aligns the calendar with the button.

accdc commented:
Awesome, thanks!

I did my best to figure out how to do this in github and gave up, it was driving me nuts, so I just cloned your folder, copied the changed files over, committed the changes and pushed them to the whatsock/bootstrap-react project.

I was confused by the one for components/intro/, because I changed the folder name "intro" to "Intro" a long time ago, and for some reason github doesn't seem to like that. When I download a fresh copy of the project though, this folder name is correct, but it's lowercase in yours which is confusing if this is a direct clone from the whatsock repo. Makes me worry a bit that all the updated files aren't being pulled into your local project, which would explain why you were getting an accordion error on your end that I couldn't produce on mine. No idea why this would be happening though.

Decrepidos commented:
Hi Bryan

I agree as I found this confusing too and could not work out what Github
was doing. I changed no folder names so I can only asume Github desktop was
automatically doing this. Again no idea.

Sorry I cause this confusion. To make sure both original and my fork are
the same I am going to completely delete my fork and re- fork. I did this
for TSG and the pull request I sent was nice a clean with no issues.

One of the things I really don’t like about Github is the convoluted way it
manages this. There seems no way to just match fork to original.

I’ll test the changes on your version

On Sat, 9 Mar 2019 at 6:58 am, Bryan Garaventa <notifications@github.com>
wrote:

> Awesome, thanks!
>
> I did my best to figure out how to do this in github and gave up, it was
> driving me nuts, so I just cloned your folder, copied the changed files
> over, committed the changes and pushed them to the whatsock/bootstrap-react
> project.
>
> I was confused by the one for components/intro/, because I changed the
> folder name "intro" to "Intro" a long time ago, and for some reason github
> doesn't seem to like that. When I download a fresh copy of the project
> though, this folder name is correct, but it's lowercase in yours which is
> confusing if this is a direct clone from the whatsock repo. Makes me worry
> a bit that all the updated files aren't being pulled into your local
> project, which would explain why you were getting an accordion error on
> your end that I couldn't produce on mine. No idea why this would be
> happening though.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/WhatSock/bootstrap-react/pull/7#issuecomment-471074090>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEWok02NVrOKEByy15FdlgSyQ09vJESfks5vUs7ygaJpZM4blMTv>
> .
>
